kano/utils/hardware.py

```diff
@@ -10,8 +10,28 @@
 from kano.utils.file_operations import read_file_contents_as_lines
 
 
-def detect_kano_keyboard():
+# "performance" scores for RPi boards
+RPI_A_SCORE = 1000
+RPI_A_PLUS_SCORE = 1000
+RPI_B_SCORE = 2000
+RPI_B_PLUS_SCORE = 2000
+RPI_ZERO_SCORE = 3000
```

Comment:
Good question! From https://en.wikipedia.org/wiki/Raspberry_Pi it mentioned that the CPU on the Zero was slightly overclocked by default (1Ghz vs 900Mhz on the B/B+). Also for the Compute module, I considered io performance since the same wikipedia page mentions it has 4 GB eMMC flash memory chip. Would you suggest changing the values?